gettsim/tests/test_zve.py

```diff
@@ -80,7 +81,7 @@ def test_zve(
     e_st_abzuege_raw_data,
 ):
     year_data = input_data[input_data["year"] == year]
-    df = year_data[IN_COLS].copy()
+    df = year_data[IN_COLS].copy()    
```

Comment:
Why trailing whitespaces?

Reply:
This should be automatically caught by the pre-commit hooks. @Eric-Sommer, can you please check that you have them turned on and they run on your local machine?